cl/recap/tasks.py

```diff
@@ -326,10 +327,6 @@ def process_recap_pdf(self, pk):
             mark_pq_status(pq, msg, PROCESSING_STATUS.QUEUED_FOR_RETRY)
             raise self.retry(exc=exc)
 
-    if not file_contents:
-        return None
```

Comment:
Well, the git blame for these two lines is not helpful, unfortunately. I guess we'll see what happens if they're missing again, but they were added in March of last year for some reason.

Reply:
Yeah, since we got rid of the `file_contents` variable it seems this shouldn't be needed anymore.